### PR TITLE
Updated my web details

### PR DESCRIPTION
**Pull request by mondus**
I have updated my personal details and set myself to Alumni. Not sure if you want the alumni option or not. Although I am not formally a team member I am still working at Sheffield and engaged in advancing the state of RSE at Sheffield and beyond.

### CODE REVIEW

File: _people/paul-richmond.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+Paul was a co-founder of the RSE team and led its growth and development from three staff to a teams of roughly 12 full time employees over a period of six years. He is the RSE team lead at the [Cambridge Institute of Computing for Climate Science](https://cambridge-iccs.github.io/) buit holds a part time role at the university of Sheffield as a Professor of Research software Engineering in the Department of Computer Science. He no longer oversees the RSE team but is actively engaged in research projects and impact work relating to the application of research software, in particular the [FLAME GPU](http://www.flamegpu.com) software for GPU accelerated agent based simulation. Paul was formally an [EPSRC Research Software Engineering Fellow](https://rse.ac.uk/community/epsrc-rse-fellows/) and President of the society of Research Software Engineering](https://society-rse.org/about/governance/rse-society-trustees-2020-2021/). He is actively engaged in the national RSE community and retains interest in advancing the national provision of RSE collaboration including within the University of Sheffield.
```
Comment (Robadob) [inline _people/paul-richmond.md]:
```suggestion
Paul was a co-founder of the RSE team and led its growth and development from three staff to a teams of roughly 12 full time employees over a period of six years. He is the RSE team lead at the [Cambridge Institute of Computing for Climate Science](https://cambridge-iccs.github.io/) but holds a part time role at the University of Sheffield as a Professor of Research Software Engineering in the Department of Computer Science. He no longer oversees the RSE team but is actively engaged in research projects and impact work relating to the application of research software, in particular the [FLAME GPU](http://www.flamegpu.com) software for GPU accelerated agent based simulation. Paul was formally an [EPSRC Research Software Engineering Fellow](https://rse.ac.uk/community/epsrc-rse-fellows/) and President of the society of Research Software Engineering](https://society-rse.org/about/governance/rse-society-trustees-2020-2021/). He is actively engaged in the national RSE community and retains interest in advancing the national provision of RSE collaboration including within the University of Sheffield. 
```

Comment (Robadob) [reply _people/paul-richmond.md]: Capitalised "University" (in uni of shef) and "Software" (in research software eng) and fixed typo "but"

Comment (ns-rse) [inline _people/paul-richmond.md]:
```suggestion
Paul was a co-founder of the RSE team and led its growth and development from three staff to a team of roughly 12 full time employees over a period of six years. He is the RSE team lead at the [Cambridge Institute of Computing for Climate Science](https://cambridge-iccs.github.io/) buit holds a part time role at the university of Sheffield as a Professor of Research software Engineering in the Department of Computer Science. He no longer oversees the RSE team but is actively engaged in research projects and impact work relating to the application of research software, in particular the [FLAME GPU](http://www.flamegpu.com) software for GPU accelerated agent based simulation. Paul was formally an [EPSRC Research Software Engineering Fellow](https://rse.ac.uk/community/epsrc-rse-fellows/) and President of the society of Research Software Engineering](https://society-rse.org/about/governance/rse-society-trustees-2020-2021/). He is actively engaged in the national RSE community and retains interest in advancing the national provision of RSE collaboration including within the University of Sheffield. 
```

`to a teams of roughly 12` > `to a team of roughly 12`

File: _people/paul-richmond.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+Paul was a co-founder of the RSE team and led its growth and development from three staff to a teams of roughly 12 full time employees over a period of six years. He is the RSE team lead at the [Cambridge Institute of Computing for Climate Science](https://cambridge-iccs.github.io/) but holds a part time role at the University of Sheffield as a Professor of Research Software Engineering in the Department of Computer Science. He no longer oversees the RSE team but is actively engaged in research projects and impact work relating to the application of research software, in particular the [FLAME GPU](http://www.flamegpu.com) software for GPU accelerated agent based simulation. Paul was formally an [EPSRC Research Software Engineering Fellow](https://rse.ac.uk/community/epsrc-rse-fellows/) and President of the society of Research Software Engineering](https://society-rse.org/about/governance/rse-society-trustees-2020-2021/). He is actively engaged in the national RSE community and retains interest in advancing the national provision of RSE collaboration including within the University of Sheffield.
```
Comment (ns-rse) [inline _people/paul-richmond.md]:
```suggestion
Paul was a co-founder of the RSE team and led its growth and development from three staff to a team of roughly 12 full time employees over a period of six years. He is the RSE team lead at the [Cambridge Institute of Computing for Climate Science](https://cambridge-iccs.github.io/) but holds a part time role at the University of Sheffield as a Professor of Research Software Engineering in the Department of Computer Science. He no longer oversees the RSE team but is actively engaged in research projects and impact work relating to the application of research software, in particular the [FLAME GPU](http://www.flamegpu.com) software for GPU accelerated agent based simulation. Paul was formally an [EPSRC Research Software Engineering Fellow](https://rse.ac.uk/community/epsrc-rse-fellows/) and President of the society of Research Software Engineering](https://society-rse.org/about/governance/rse-society-trustees-2020-2021/). He is actively engaged in the national RSE community and retains interest in advancing the national provision of RSE collaboration including within the University of Sheffield. 
```

**Pull request merged**